apps/src/lab2/levelEditors/aichatSettings/EditAichatSettings.tsx

```diff
@@ -88,13 +89,31 @@ const EditAichatSettings: React.FunctionComponent<{
     [aichatSettings, setAichatSettings, initialCustomizations]
   );
 
+  const setModelSelectionValues = useCallback(
+    (additionalModelIds: Set<string>, selectedModelId: string) => {
```

Comment:
Similar to the previous comment - can `additionalModelIds` just be a string array? Looks like we create a new `Set` out of it anyway which should remove duplicates